broadway_api/utils/bootstrap.py

```diff
@@ -3,44 +3,85 @@
 import logging
 import os
 import sys
-import uuid
+import signal
+
+from typing import Dict, Any
+from queue import Queue
 
 from pymongo import MongoClient
 from pymongo.errors import ConnectionFailure
 
+import tornado
+import tornado.web
+import tornado.ioloop
+import tornado.httpserver
+
+from logging.handlers import TimedRotatingFileHandler
+
 from broadway_api.definitions import course_config
 from broadway_api.daos import CourseDao, WorkerNodeDao
 from broadway_api.models import Course
 
-import config
+import broadway_api.callbacks as callbacks
+import broadway_api.handlers.client as client_handlers
+import broadway_api.handlers.worker as worker_handlers
+import broadway_api.handlers.worker_ws as worker_ws_handlers
+
+logger = logging.getLogger("broadway.bootstrap")
+
+
+def initialize_global_settings(flags: Dict[str, Any]) -> Dict[str, Any]:
+    return {
+        "CLUSTER_TOKEN": flags["token"],
```

Comment:
Why save the cluster token separately when it already exists in `flags` which is also saved. We can replace all instances of `settings["CLUSTER_TOKEN"]` to `settings["FLAGS"]["token"]`.